### PR TITLE
fix #74902 robot-forum.com

### LINKED ISSUE #74902: robot-forum.com
### Issue URL (Anti Adblock Script)
 [https://www.robot-forum.com/robotforum/thread/11112-converting-tp-programs-to-text-file/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.robot-forum.com%2Frobotforum%2Fthread%2F11112-converting-tp-programs-to-text-file%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v        1.8.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by wout-codes**
#74902

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Hi! Anything that I can add for `content_blocker.txt`?

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -545,6 +545,8 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74902
+ robot-forum.com#%#//scriptlet("abort-current-inline-script", "$", "offsetHeight")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:549]: Re-write the rule a bit `robot-forum.com#%#//scriptlet("abort-current-inline-script", "$", "adblock")`

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:549]: Why though?

**Comment by zloyden** [comment]
Hi! For content blockers
```
! https://github.com/AdguardTeam/AdguardFilters/issues/74902
@@||robot-forum.com^$generichide
@@||robot-forum.com/*/adsbygoogle.js$domain=robot-forum.com
```

**Comment by zloyden** [comment]
There is also a leftover `robot-forum.com#?#.boxContainer > div.box ins.adsbygoogle:upward(.boxContainer)`

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: We should use simple and clear rules, if possible.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: Okay.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+robot-forum.com#?#.boxContainer > div.box ins.adsbygoogle:upward(.boxContainer)
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:257]: Rules with `#?#` always are to the `css_extended.txt`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by AdamWr** [comment]
On my end `abort-current-inline-script` rule causes that search works differently than without ad blocker.
With `abort-current-inline-script` it looks like this:
<details><summary>Screenshot</summary>

image

</details>

With disabled rule `robot-forum.com#@%#//scriptlet("abort-current-inline-script", "$", "adblock")`:
<details><summary>Screenshot</summary>

image

</details>

What about changing it to something like:
`robot-forum.com#%#//scriptlet("set-constant", "cwAdblockDisabled", "true")`

**Comment by wout-codes** [comment]
@AdamWr I don't have problems but You can swap the rules.

**Comment by AdamWr** [comment]
Ok, done.

**Pull request merged**